File: backend/ApiServer/src/main/java/ar/com/tandilweb/ApiServer/dataTypesObjects/users/UserProfile.java
package ar.com.tandilweb.ApiServer.dataTypesObjects.users;

import ar.com.tandilweb.ApiServer.dataTypesObjects.generic.GeneralResponse;

public class UserProfile extends GeneralResponse {
	
	public long idUser;
	public String nick;
	public String photo;
	public long chips;
	public String validation_code;
	
}

File: frontend/newClient/src/app/services/UserProfile.ts
import { Room } from './roomsResponse';

export class UserProfile {
  public idUser: number;
  public nick: string;
  public photo: string;
  public chips: number;
}

export class FriendCard {
  public idUser: number;
  public nick: string;
  public photo: string;
  public inGame: boolean;
  public rooms: Room[];
  public status: string;
}

export class FriendData {
  public friends: FriendCard[];
}
